app/config/api.py

```diff
@@ -1,11 +1,10 @@
 from ninja import NinjaAPI
 from provider.api import router as provider_router
 
-root = NinjaAPI()
 api = NinjaAPI()
 
 
-@root.get('/checker')
+@api.get('/checker')
```

Comment:
:thinking: We register the `api` object in `urls.py` as

```py
urlpatterns = [
    path('api/', api.urls),
    path('admin/', admin.site.urls),
]
```

Doesn't that mean, that with this change the checker is reachable at `api/checker` instead of `/checker`?

Reply:
I see three reasons for putting it under root instead of `/api`:
1. In service-stac, we have it under root [if I understand correctly](https://github.com/geoadmin/service-stac/blob/8566c53d0300013a2ffe2de8011229d26ec616c0/app/config/urls.py#L33) (`/checker`)
2. For [Kubernetes pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/), usually it is under root (`/healthz`).
3. As a beginner, I would intuitively expect stuff to interact with under `/api`. Infrastructure-related endpoints would be somewhere else, for example under root.

But no strong opinion. If it's too much of a hassle to change or if you prefer `/api/checker`, I am ok with keeping it.

What is your preference?

Reply:
I agree! I'm gonna change it ASAP